phpBB/phpbb/di/container_builder.php

```diff
@@ -420,9 +434,14 @@ protected function load_extensions()
 				$filename = $path . 'vendor/autoload.php';
 				if (file_exists($filename))
 				{
-					require $filename;
+					$autoloaders .= "require '{$filename}';\n";
 				}
 			}
+
+			$configCache = new ConfigCache($this->get_autoload_filename(), false);
```

Comment:
I'm using ConfigCache just to not have to write the dumpFile logic myself but I don't need the others features (including debug versus not debug)